### DISCUSSION TITLE
Old goreleaser version in $GOPATH/bin can result in unusable goreleaser on Mac

### DISCUSSION THREAD

**Discussion by KevinGimbel** [goreleaser/goreleaser#3226] [Q&A] [answered]
I'm not sure where to report this, but I just encountered a weird issue: I've had an old version of goreleaser from 2017 or 2018 in my `$GOPATH/bin` directory that I forgot about, when I tried to install it with brew and `go install` I'd always get the following issue:

```shell
fish: Job 1, 'goreleaser --version' terminated by signal SIGSEGV (Address boundary error)
```

I was about to create an issue when I tried to call some other Go binaries directly from `$GOPATH/bin` and notices the super old binary I still had.

As I said, I'm unsure where to put this information but I feel like there may be others that have similar issues, so I wanted to document it somewhere.

**Comment by KevinGimbel** [comment, marked as answer]
I guess the main problem here is the switch in architecture, as my new Mac has the M1 chip and the one I used back in 2017/2018 had an intel chip.

**Reply by caarlos0** [reply]
yes, that seems likely the issue.

nothing we can do about it... 
you can of course delete any old binaries with something like `rm $(which goreleaser)`.
